src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
+  const liveService = _getLiveService();
+  const activeUser = User.getActiveUser();
+
+  return liveService.registerUser(activeUser)
+    .then((pubnubConfig) => {
+      const pubnubClient = new PubNub(pubnubConfig);
+      const listener = new PubNubListener();
+      liveService.initialize(pubnubClient, listener);
+    });
+}
+
+function unregister() {
+  const liveService = _getLiveService();
+  return liveService.unregisterUser()
+    .then((resp) => {
+      liveService.uninitialize();
```

Comment:
I prefer to keep `LiveService`'s methods as simple as possible and organize more complex operations in `LiveServiceFacade`. Like `LiveServiceFacade.register()` takes care of registering the user and the initializing `LiveService`. `LiveServiceFacade.unregister()` unregisters the user and uninitializes `LiveService`. Does that make sense?